notebooks/cookbook/Enhanced sampling methods/Umbrella sampling MD.ipynb

```diff
@@ -0,0 +1,436 @@
+    "                                                                  "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# define a function to list force groups and modify system context                                                       \n",
+    "def forcegroupify(system):                                        \n",
+    "    forcegroups = {}                                              \n",
+    "    for i in range(system.getNumForces()):                                                                                           \n",
+    "        force = system.getForce(i)                           \n",
+    "        force.setForceGroup(i) \n",
+    "        forcegroups[force] = i                                    \n",
+    "    return forcegroups"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Create system + CV Forces with harmonic restraints"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sim_temp = 300.0 * kelvin\n",
+    "H_mass = 4.0 * amu #Might need to be tuned to 3.5 amu \n",
+    "time_step = 0.002 * picosecond  \n",
+    "nb_cutoff = 10.0 * angstrom                                                                                                          \n",
+    "box_padding = 12.0 * angstrom\n",
+    "salt_conc = 0.15 * molar\n",
+    "receptor_path=\"../villin.pdb\"\n",
+    "current_file=\"villin-solvated\"\n",
+    "# Misc parameters                                                 \n",
+    "restraint_distance = 0.0 * angstroms                              \n",
+    "restart_freq = 10\n",
+    "log_freq = 1                                                                                                                         \n",
+    "prd_steps = 100     "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "5"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "## Load an already solvated PDB file and set up the system + state\n",
+    "pdb = PDBFile(receptor_path)\n",
+    "omm_forcefield = ForceField(\"amber/ff14SB.xml\", \"amber14/tip3p.xml\")\n",
+    "system = omm_forcefield.createSystem(pdb.topology, \n",
+    "                                         nonbondedMethod=PME, \n",
+    "                                         nonbondedCutoff=nb_cutoff, \n",
+    "                                         constraints=HBonds, \n",
+    "                                         rigidWater=True,\n",
+    "                                         hydrogenMass=H_mass)\n",
+    "system.addForce(MonteCarloBarostat(1*bar, sim_temp))\n"
```

Comment:
Adding a barostat is unnecessary.  It's a fine thing to do in a real simulation, but it's unrelated to umbrella sampling.  It doesn't contribute toward showing the user what we're trying to show them.